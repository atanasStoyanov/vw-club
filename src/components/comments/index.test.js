import React from 'react';
import renderer from 'react-test-renderer';
import Comments from '.';
jest.mock('../comment', () => 'Comment');

describe('Comments Component', () => {
    it('should return "No comments yet"', () => {
        const post = {
            comments: []
        }
        const tree = renderer.create(<Comments post={post}/>)
        expect(tree).toMatchSnapshot();
    })

    it('should return one comment', () => {
        const post = {
            comments: [{
                _id:1
            }]
        }
        const tree = renderer.create(<Comments post={post}/>)
        expect(tree).toMatchSnapshot();
    })
})
