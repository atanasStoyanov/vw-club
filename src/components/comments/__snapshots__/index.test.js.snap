// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Component should return "No comments yet" 1`] = `
<h3>
  No comments yet!
</h3>
`;

exports[`Comments Component should return one comment 1`] = `
<Comment
  _id={1}
/>
`;
